Firmware/LowLevel/src/main.cpp

```diff
@@ -372,9 +369,7 @@ void setup() {
     //  Therefore, we pause the other core until setup() was a success
     rp2040.idleOtherCore();
 
-#ifdef USB_DEBUG
-    DEBUG_SERIAL.begin(9600);
-#endif
+    DEBUG_BEGIN(9600);
```

Comment:
I think that's no longer defined now...?

Reply:
Sure, that's how Clemens made the uart debugging. My one was only an extension so that I do not write all the #ifdef's and that stuff :wink:

Reply:
Sure, but wasn't `DEBUG_BEGIN` only defined in `debug.h`, which is now gone?
Sorry, I didn't pull your latest branch and try to compile it, therefore it's just a review from looking at the code. I've been trying to train an AI model for the past few days, still not working... :/

Reply:
That's how it look now:
```
#ifdef USB_DEBUG
    DEBUG_SERIAL.begin(9600);
#endif
```
Look like you're in the wrong files